python/py-fonttools/Portfile

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
+
+name                py-fonttools
+version             4.26.2
```

Comment:
according to upstream, this version doesn't support Python 2.7, so i you still need the PY27 subport that one should likely be restricted to the 3.x branch.

Reply:
It already is: https://github.com/macports/macports-ports/pull/12150/files#diff-75dfe2fa780c961bc28af98945fd6ef85533318499239b8aec011578870c2e36R37-R43

Reply:
oh, I am sorry @amake ! I should have some coffee first ;)